### DISCUSSION TITLE
Error while  running the ansible front-end script at Creating Nginx dir

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1606] [Q&A] [answered]
Hi Team,

while I'm running the front-end ansible script, I was getting the below error at create nginx directory Task.
workstation public IP: 50.16.121.96
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/935686bf-86d2-42ba-b601-443f38d0386b)

**Comment by sreedharm07** [comment, marked as answer]
Hi @omswetha 
in your frontend.yml it supposed to be **directory** instead of present. i changed in your machine its worked fine, please try once.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/180034b6-cad3-4274-b29f-cebd6d02d67a)

**Reply by omdevops99** [reply]
Thanks @sreedharm07
